input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -353,7 +353,8 @@ And, the equivalent with nested structure:
 Note the former example is preferred due to its simplicity and the latter is included purely for illustrative purposes.
 
 ### Column Ordering
-*View runners* MUST produce output in the order that the columns appear in the ViewDefinition.
+
+*View runners* that support column ordering in their output format MUST order the columns of the result to match the order that they appear in the ViewDefinition.
```

Comment:
I support this qualification, but I don't know about the original language "match the order that they appear". If the view definition is JSON, we don't require the keys to appear in any specific order..

What do you think of "match the order defined here" (i.e., in this section of the spec)?

Reply:
Agreed, fixed in fe6cb5cfb8971534f7c801caaf8aaaf57942a9e6